src/utils/mk64.py

```diff
@@ -0,0 +1,82 @@
+from src.utils import uploader
+
+
+def compare_records(original, newrecords):
+    racer_ids = get_racer_ids()
+    slugs = get_tracklist('slugs')
+
+    for j, (old_times, new_times) in enumerate(zip(original, newrecords)):
+        for i, (or_re, nr_re) in enumerate(zip(old_times['track-record'].records, new_times['track-record'].records)):
+            if i > 4:
+                break
+            if or_re != nr_re:
+                # Commented these out because not being used atm
+                # track_name = old_times['name']
+                # racer_name = racer_ids[nr_re['character']]
+                slug = slugs[j]
+                new_time = nr_re['time']
+                uploader.send_to_gus(new_time, slug, rtype="3lap")
+                break
+
+        if old_times['track-record'].records[5] != new_times['track-record'].records[5]:
+            # Commented these out because not being used atm
+            # track_name = old_times['name']
+            # racer_name = racer_ids[nr_re['character']]
+            new_time = new_times['track-record'].records[5]['time']
+            uploader.send_to_gus(new_time, slugs[j], rtype='flap')
+
+
+def get_racer_ids():
```

Comment:
This pattern intrigued me. Is there a reason you want this list to be behind a function call?

Here is what I could think of:

* Putting it behind a function call, instead of being a top-level list means that we always get the same list when we perform the function call, it can't accidentally be mutated under us.

This seems like a good thing, so _if that's the reason_, my proposal is to make it an immutable value, but still have it be a top-level value.

If that's not the reason, I'd like to understand the motivation, since this pattern (used below as well) is new to me.

Reply:
This is mostly because I am not sure how to export top level values in Python, and I was already having enough trouble with the modules. If there is an easy way to do it, then yeah I agree that seems more intuitive.

Another option is converting mk64 to a class, and then have the racer ids, tracks, etc... be class variables. Up to you guys.

Reply:
OK - I was way overthinking it. I didn't realize that you just have to declare the variables and Python automatically exports them (pretty nice, actually).

I made a separate file, data/game.py, which includes all the relevant game data which can be imported into the other modules as needed, lmk what you guys think.

Reply:
Nice!